src/db/vco_effects.loc.tsv

```diff
@@ -16,3 +16,4 @@ effects_description_vco_ogr_greasus_speed	Speed: %+n for Greasus Goldtooth	true
 effects_description_vco_ogr_greasus_charge_bonus	Charge Bonus: %+n for Greasus Goldtooth	true
 effects_description_vco_ogr_greasus_armour	Armour: %+n for Greasus Goldtooth	true
 effects_description_vco_ogr_greasus_ward_save	Ward save: %+n% for Greasus Goldtooth	true
+effects_description_vco_skv_pooled_resource_skaven_food_victory	Food rewarded for Victory Conditions	true
```

Comment:
Do not mention "Victory Conditions" in the flavour. It should also be more verbose by including the actual ammount: `Food: %+n per turn`